docs/src/components/DropDownExample/DropDownExample.component.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * This component is used at the top of dropdown.mdx to immediately show an example of what
+ * a card looks like. This same example is given again at the bottom of the page but with the
+ * live code editor.
+ */
+
+import * as React from 'react';
+import { colors, DropDown } from '@retailmenot/anchor';
+import { MyList } from './MyList.component';
+import { MouseOverMe } from './MouseOverMe.component';
+
+export const DropDownExample = () => (
+    <DropDown overlay={<MyList />} position={{ left: 0 }}>
```

Comment:
I do wish that the List was part of the live example, so that I could modify the styling for that as well.

Reply:
I agree. I haven't figured out a good way to go about doing that yet; this is a baby step towards it.